libs/mstable/explore/src/components/CoreVaultCard/components/DefaultVaultCard.tsx

```diff
@@ -76,54 +79,88 @@ export const DefaultVaultCard = (props: CoreVaultCardProps) => {
       >
-        <ValueLabel
-          label={apyLabel}
-          hint={apyHint}
-          components={{
-            valueContainer: { pb: 0.3 },
-            label: { sx: { mb: 0.5 } },
-          }}
-        >
-          <Typography variant="value3">
-            {isLoading ? <Skeleton width={50} /> : apy}
-          </Typography>
-        </ValueLabel>
-        <ValueLabel
-          label={balanceLabel}
-          hint={balanceHint}
-          components={{
-            valueContainer: { pb: 0.3 },
-            label: { sx: { mb: 0.5 } },
-          }}
-        >
-          <Typography variant="value3">
-            {isLoading ? <Skeleton width={50} /> : balance}
-          </Typography>
-        </ValueLabel>
-        <ValueLabel
-          label={tvlLabel}
-          hint={tvlHint}
-          components={{
-            label: { sx: { mb: 0.5 } },
-          }}
-        >
-          <Stack direction="row" spacing={1} alignItems="baseline">
+        <Grid2 xs={6}>
+          <ValueLabel
+            label={apyLabel}
+            hint={apyHint}
+            components={{
+              valueContainer: { pb: 0.3 },
+              label: { sx: { mb: 0.5 } },
+            }}
+          >
             <Typography variant="value3">
-              {isLoading ? <Skeleton width={50} /> : tvl}
+              {isLoading ? <Skeleton width={50} /> : apy}
             </Typography>
-          </Stack>
-        </ValueLabel>
-      </Stack>
+          </ValueLabel>
+        </Grid2>
+        <Grid2 xs={6}>
+          <ValueLabel
+            label={tvlLabel}
+            hint={tvlHint}
+            components={{
+              label: { sx: { mb: 0.5 } },
+            }}
+          >
+            <Stack direction="row" spacing={1} alignItems="baseline">
+              <Typography variant="value3">
+                {isLoading ? <Skeleton width={50} /> : tvl}
+              </Typography>
+            </Stack>
+          </ValueLabel>
+        </Grid2>
+        <Grid2 xs={6}>
+          <ValueLabel
+            label={balanceLabel}
+            hint={balanceHint}
+            components={{
+              valueContainer: { pb: 0.3 },
+              label: { sx: { mb: 0.5 } },
+            }}
+          >
+            <Typography variant="value3">
+              {isLoading ? <Skeleton width={50} /> : balance}
+            </Typography>
+          </ValueLabel>
+        </Grid2>
+        <Grid2 xs={6}>
+          <ValueLabel
+            label={roiLabel}
+            components={{
+              valueContainer: { pb: 0.3 },
+              label: { sx: { mb: 0.5 } },
+            }}
+          >
+            {isLoading ? (
+              <Skeleton width={50} />
+            ) : (
+              <Typography
+                variant="value3"
+                color={
+                  formattedRoiUsd === '0'
+                    ? undefined
+                    : +formattedRoiUsd > 0
+                    ? 'success.main'
+                    : 'error.main'
```

Comment:
quite a lot of transformations. Seems like roiUsd prop as number from useCoreVaultCardProps would be more simple to handle.
also $ sign can be moved to Intl.NumberFormat with all the other formatting options